docs/general/ad-filtering/create-own-filters.md

```diff
@@ -1007,6 +1026,8 @@ as [$document exclusion](#document-modifier) — unblock everything.
 
 Rules with `$urlblock` modifier are not supported by AdGuard Content Blocker.
 
+The `$urlblock` modifier is supported but not implemented in AdGuard Browser Extension MV3.
```

Comment:
This again, "supported but not implemented" makes no sense

Reply:
@SeverAnna 
so here we should delete restriction because we will implement it (someday)